### PR TITLE
Fixes xenomorphs being able to hold any item

### LINKED ISSUE #69615: xenos have fully working hands
Reporting client version: 514.1585

## Round ID:
[189617](https://scrubby.melonmesa.com/round/189617)

## Testmerges:
- [Converts SSblackbox feedback to an associative list](https://www.github.com/tgstation/tgstation/pull/69399)

## Reproduction: be a xeno pick up a gun have fun better then shitty hardstun shoves 

image

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Advanced trait user was moved into the brain, which meant Xenomorphs (who have brains) were now advancedtoolusers, and as such were able to manipulate non-xeno stuff.

## Why It's Good For The Game

Closes https://github.com/tgstation/tgstation/issues/69615

## Changelog

:cl:
fix: Xenomorphs can no longer hold any item in the game,
/:cl:

### CODE REVIEW

**Comment by theta060** [comment]
now hear me out remove all beno hardstun  and keep the high functioning trait in now they are a lot more fun to play and to fight

File: code/modules/mob/living/brain/brain_item.dm [hunk 1]
```diff
@@ -323,6 +323,7 @@
 	name = "alien brain"
 	desc = "We barely understand the brains of terrestial animals. Who knows what we may find in the brain of such an advanced species?"
 	icon_state = "brain-x"
+	organ_traits = list(TRAIT_CAN_STRIP, TRAIT_LITERATE)
```
Comment (timothymtorres) [inline code/modules/mob/living/brain/brain_item.dm:326]:
```suggestion
	organ_traits = list(TRAIT_CAN_STRIP)
```

Xenomorphs should not be able to read or write.

Comment (JohnFulpWillard) [reply code/modules/mob/living/brain/brain_item.dm:326]: from science fiction, though I couldn't find a definitive answer, xenomorphs seems to be pretty smart.

**Comment by Acer202** [comment]
> now hear me out remove all beno hardstun and keep the high functioning trait in now they are a lot more fun to play and to fight

NT vs Xenos baseball game when?

**Comment by Palpetinus** [comment]
"The fucking XENO has a GUN!"

File: code/modules/mob/living/brain/brain_item.dm (hunk 1 above)
Comment (timothymtorres) [reply code/modules/mob/living/brain/brain_item.dm:326]: I played too much Colonial Marines back in the day, but I feel like this breaks the traditional xenomorph lore and flavor.

When I made the PR for literacy, I specifically was aiming to make it granted _exclusively_ to human subtypes. (with a few rare exceptions)  I really don't feel like xenomorphs should be included into that category, but if you are deadset on this, then at least limit it to only xeno queens.

**Comment by itseasytosee** [comment]
My oversight, but anyways while you are down there you should probably remove the xeno's innate TRAIT_CAN_STRIP trait since they get it from there brain

**Comment by msgerbs** [comment]
Is this why everybody has been doing xeno investor and such admemes all day?

**Comment by theta060** [comment]
> My oversight, but anyways while you are down there you should probably remove the xeno's innate TRAIT_CAN_STRIP trait since they get it from their brain

xenos need this to strip suits so they can face hug people

**Comment by itseasytosee** [comment]
> > My oversight, but anyways while you are down there you should probably remove the xeno's innate TRAIT_CAN_STRIP trait since they get it from their brain
> 
> xenos need this to strip suits so they can face hug people

Yeah but there brain gives it to them

**Pull request merged**